spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -284,6 +284,7 @@ content into your application. Rather, pick only the properties that you need.
 	server.tomcat.resource.cache-ttl= # Time-to-live of the static resource cache.
 	server.tomcat.uri-encoding=UTF-8 # Character encoding to use to decode the URI.
 	server.tomcat.use-relative-redirects= # Whether HTTP 1.1 and later location headers generated by a call to sendRedirect will use relative or absolute redirects.
+	server.tomcat.processor-cache= # Custom value for Tomcat's processor cache.
```

Comment:
Properties are ordered in natural order. This property must be moved. 

Also, the field description must match the description here.